bin/ch/stdafx.h

```diff
@@ -30,6 +30,7 @@
 #ifdef _WIN32
 #include <windows.h>
 #else
+#include <string>
```

Comment:
Consider move this to line 29 and share for all, and remove the `#include` from `WScriptJsrt.cpp`.